packages/solo/src/scratch.js

```diff
@@ -2,25 +2,27 @@ import { Far } from '@agoric/marshal';
 
 export default function makeScratchPad() {
   const map = new Map();
-  async function get(idP) {
```

Comment:
I don't see any substantive change here. Drive-by refactor? Leave a note in the commit comment in the future, please.

Reply:
The substantive change was adding `init`.

Reply:
I also added `keys` and `delete` for better Store-like compatibility.

Reply:
ah. I somehow couldn't see it. thanks.